charts/opensearch/README.md

```diff
@@ -82,7 +82,7 @@ helm uninstall my-release
 | `roles`                            | A list of the specific node [roles][] for the `nodeGroup`                                                                                                                                                                                                | see [values.yaml][]                             |
 | `schedulerName`                    | Name of the [alternate scheduler][]                                                                                                                                                                                                                       | `""`                                            |
 | `secretMounts`                     | Allows you easily mount a secret as a file inside the StatefulSet. Useful for mounting certificates and other secrets. See [values.yaml][] for an example                                                                                                 | `[]`                                            |
-| `securityConfig`                   | Configure the opensearch security plugin. There are multiple ways to inject configuration into the chart, see [values.yaml](https://github.com/opensearch-project/helm-charts/blob/main/charts/opensearch/values.yaml) details.                           | By default an insecure demonstration configuration is set. This **must** be changed before going to production. |
+| `securityConfig`                   | Configure the opensearch security plugin. There are multiple ways to inject configuration into the chart, see [values.yaml][] details.                           | By default an insecure demonstration configuration is set. This **must** be changed before going to production. |
```

Comment:
Not sure why you change it to `[values.yaml][]` can you provide the actual working link? Thanks.

Reply:
The actual working link is in reference section, see line #112 
I've changed it here to `[]` just to avoid repeating the URL.

Reply:
Got it.